lib/matplotlib/tests/test_contour.py

```diff
@@ -557,15 +557,15 @@ def test_find_nearest_contour_no_filled():
     cs = plt.contourf(img, 10)
 
     with pytest.warns(mpl._api.MatplotlibDeprecationWarning), \
```

Comment:
We should no longer get the warning from the `collections` attribute, but there is still a deprecation on `find_nearest_contour` itself.  Since #27075 removes that, I thought I would leave it so as not to duplicate effort.

Having said that, I have no objection to pulling @jklymak's change in here, and it may be better to do so, so that we can see the warning disappear.

Reply:
We probably should, to have this all in one PR discussion.

Reply:
Yes feel free to take over my PR!

Reply:
Done